My take on Exercise 9.3.5 (#1)
howardbaek/mastering-shiny-book-exercises-solutions-book: 09-uploads-downloads.Rmd

knitr::opts_chunk$set(eval = FALSE)
```

> ## 9.4 Exercises {.unnumbered}

Reviewer: Is {.unnumbered} the same as {-}?

Author: I dont remember I did this, it is likely got edited automatically when I switched to "visual markdown editor" mode in rstudio. Do you want me to change it back?

Author: Yea, I have changed it back.

shinyApp(ui, server)
```

> 5.  From 9.3 Case study, the main changes is from the Clean step of server function, where one large reactive is broken down into three multiple smaller ones.

Reviewer: ```suggestion
5.  From the 9.3 Case study, the main change happens in the cleaning step inside the server function, where one large reactive is broken down into three smaller ones.
```
